frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/editCommercialDetails.jspf

```diff
@@ -65,6 +87,8 @@
                         <core_rt:if test="${release.cotsDetails.ossContractSigned == 'true'}"> checked="checked" </core_rt:if>
                 /> OSS contract signed</label>
```

Comment:
This is only a small one. The first letter should be in uppercase like in all the other labels.
OSS Contract Signed